src/pages/contact/contact.html

```diff
@@ -20,7 +20,7 @@
   <ion-list>
     <ion-item *ngFor="let contact of contacts" (click)="showContactModal(contact)">
       <h2>{{contact.name}}</h2>
-      <p>{{contact.phone_number | phone}}</p>
+      <p>{{contact.phone_number}}</p>
```

Comment:
the phone numbers are piped in the back end now and so this was redundant. Nice find.